TESTAUTO.md

```diff
@@ -83,14 +85,14 @@ This should setup the test data and then run a test that verifies the data persi
     * Test class: ```org.orcid.integration.blackbox.BlackBoxTestSuite```
     * Test runner: ```JUnit 4```
 
-6. In the Arguments tab, set the following VM arguments
+6. In the Arguments tab, set the following VM arguments (note that you need to insert your `webdriver.gecko.driver` path)
```

Comment:
I've now suggested ~/bin for installation, but am not pre-populating the full path because we don't know the users home directory.